StringOps.java

```diff
@@ -22,21 +22,112 @@ public class StringOps {
     //////                                               ///////
     ////////////////////////////////////////////////////////////
     public static void main(String[] args) {
-        
+        System.out.println(capVowelsLowRest("Hello World"));
+        camelCase("   Intro  to   coMPUter    sCIEncE ");
+        allIndexOf("Hello world",'l'); // output: {2, 3, 9}
     }
 
     public static String capVowelsLowRest (String string) {
-        // Write your code here:
-        return "";
+        String vowels = "aeiou";
+        String capVowels = "AEIOU";
+        String withBigVowels = "";
+        boolean isVowel = false;
+        for (int i = 0; i < string.length(); i++) {
+            for (int j = 0; j < vowels.length(); j++) {
+                if (string.charAt(i) == vowels.charAt(j)) {
+                    withBigVowels += (char) (string.charAt(i) - 32);
+                    isVowel = true;
+                } else if (string.charAt(i) == capVowels.charAt(j)) {
+                    withBigVowels += string.charAt(i);
+                    isVowel = true;
+                }
+            }
```

Comment:
No need loop for vowels search...
```suggestion
            char currentChar = string.charAt(i);
            if (vowels.indexOf(currentChar) != -1) {
                withBigVowels += (char) (string.charAt(i) - 32);
            } else if (capVowels.indexOf(currentChar) != -1) {
                 withBigVowels += string.charAt(i);
             }
```
Check next comments to understand why isVowel removed.